acct.bionimbus.org/manifest.json

```diff
@@ -8,24 +8,23 @@
     "autodeploy": "yes"
   },
   "versions": {
-    "ambassador": "quay.io/datawire/ambassador:1.4.2",
```

Comment:
This line got deleted. Is this important though?

Reply:
Looks like workspaces were never enabled properly in `gitops-qa`, we can either:
1. Just push a commit to avoid this deletion
2. Introduce ambassador to `https://github.com/uc-cdis/gitops-qa/blob/master/qa-acct.planx-pla.net/manifest.json` and run the automation again to replicate all changes and create a new PR.

Reply:
right, we (@atharvar28 and I) removed the workspaces from QA because it wasn't needed. So it's a catch now... 
I added another commit to keep ambassador here in prod :)